lib/data/find-eu-exit-guidance-business.yml

```diff
@@ -13,134 +13,194 @@
+        :title: Oil, gas and coal
+        :value: oil-gas-coal
+      - :content_id: 2c562ab5-7891-4a00-8510-a5ebdb4001c8
+        :title: Other energy
+        :value: other-energy
+      - :content_id: 7536c0c4-fb41-43f4-a2c4-08f4fa9f5427
+        :title: Other manufacturing
+        :value: other-manufacturing
       - :content_id: d9da7c7b-b0c9-4757-aa45-9c70dd78df62
-        :title: Other personal services
-        :value: other-personal-services
-      - :content_id: 9c0d1627-0423-4159-a947-e6abab0d0afd
-        :title: Parts and machinery
-        :value: parts-and-machinery
+        :title: Personal services
+        :value: personal-services
       - :content_id: 93f5c156-2449-4e52-9e92-aefc73586ffd
         :title: Pharmaceuticals
         :value: pharmaceuticals
+      - :content_id: 4591ce75-7568-48c6-af5f-3558f89f7f57
+        :title: Ports and airports
+        :value: ports-airports
       - :content_id: 16a9ce39-7786-4ae2-8289-93d924c958a5
-        :title: Post
-        :value: post
+        :title: Postal and courier services
+        :value: postal-courier-services
       - :content_id: f07527f1-e6de-4fe8-b7d9-b197769e1f1f
         :title: Professional and business services
         :value: professional-and-business-services
       - :content_id: '091c0e83-bb54-4727-9b2c-a919510e1e79'
-        :title: Public administration and defence
-        :value: public-administration-and-defence
+        :title: Public administration
+        :value: public-administration
       - :content_id: 53bcfcf0-e22a-416d-91ac-2661e1d6ed04
         :title: Rail
         :value: rail
+      - :content_id: 2e1cf5a9-d410-4cfc-9f54-d94302f8fbd1
+        :title: Rail (passengers and freight)
+        :value: rail-passenger-freight
       - :content_id: dba1b198-1ead-4de4-a961-b11d71fc6f20
-        :title: Real estate (excluding imputed rent)
+        :title: Real estate
         :value: real-estate-excl-imputed-rent
+      - :content_id: c1d8057c-76bf-431c-9ac8-6281a4b7b9ca
+        :title: Repair of computers and consumer goods
+        :value: repair-of-computers-consumer-goods
+      - :content_id: 39771bc2-8b2c-43e9-9cfc-fa3606b67db6
+        :title: Research
+        :value: research
+      - :content_id: a572f853-18bf-41b6-9afa-82c1723e3de9
+        :title: Restaurants, bars and catering
+        :value: restaurants-bars-catering
       - :content_id: 34a6edd0-46ea-4a76-ae80-8c96709d4f59
-        :title: Retail
+        :title: Retail and wholesale (excluding motor trade, food and drink)
         :value: retail
-      - :content_id: a9736af7-4e8b-4b1f-9a4b-9482252f4a4f
-        :title: Social work
-        :value: social-work
-      - :content_id: 99eeb13a-ce9d-49f6-aa5b-68d0e4a45358
-        :title: Steel and other metals or commodities
-        :value: steel-and-other-metals-commodities
+      - :content_id: 6f5f8dbb-3cd3-475d-8f3a-ed6e22d370be
+        :title: Road (passengers and freight)
+        :value: road-passenger-freight
```

Comment:
My mistake, this should be `road-passengers-freight`. I'll update my PRs to reflect that too.